modules/wallet/encrypt.go

```diff
@@ -122,6 +122,12 @@ func (w *Wallet) managedUnlock(masterKey crypto.TwofishKey) error {
 		// lastChange
 		lastChange = dbGetConsensusChangeID(w.dbTx)
 
+		// currentHeight
+		currentHeight, err = dbGetConsensusHeight(w.dbTx)
```

Comment:
this is definitely worth testing, since it's the only place where `scanHeight` doesn't start from 0